src/main/java/com/internetitem/logback/elasticsearch/config/Settings.java

```diff
@@ -23,6 +23,7 @@
 	private int maxQueueSize = 100 * 1024 * 1024;
 	private Authentication authentication;
 	private int maxMessageSize = -1;
+    private boolean enableContextMap;
```

Comment:
Your indentation in this MR seems to be off (inconsistent use of tabs and spaces)